public/index.html

```diff
@@ -25,6 +25,14 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Breads</title>
+    <!-- Below JSDelivr CDN and Widget script for DarkMode -->
+    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js" defer></script>
+    <script>
```

Comment:
Please add `defer` to this script tag too

Reply:
Done. 
Thank you and all the gang in the back for the opportunity of doing this.
Gonna try to pick up the next one, though I'm new and everything looks confusing.